esmvalcore/cmor/table.py

```diff
@@ -73,6 +73,8 @@ def __init__(self, cmor_tables_path, default=None):
         cmor_tables_path = self._get_cmor_path(cmor_tables_path)
 
         self._cmor_folder = os.path.join(cmor_tables_path, 'Tables')
+        if 'obs4mips' not in cmor_tables_path:
```

Comment:
This will not work if another project is created that also has no controlled vocabulary file available. I think it would be better to check if there is a controlled vocabulary file, e.g.
```
if Path(self._cmor_folder).glob('*_CV.json'):
```

Reply:
obs4mips has a CV file but its contents are different that the CMIP6 CV file. The thing here is that I only want to load it for CMIP6 because everything else that I do later with the activities and institutes would not work for obs4mips.

Reply:
That is unfortunate. In that case, could you change the code to make it read the institutes and activities if the information is present in the correct format (i.e. as in CMIP6) and just leave it empty if it is not there or not in the correct format? It should be possible to add additional or different tables without modifying the code, as this is actually a feature that we support (you can specify a custom config-developer.yml file in your config-user.yml file, defining additional projects and tables).